chat.js

```diff
@@ -916,8 +916,13 @@ Chat.loadCommands = function () {
 	Object.assign(commands, require('./chat-plugins/info').commands);
 
 	for (let file of fs.readdirSync(path.resolve(__dirname, 'chat-plugins'))) {
-		if (file.substr(-3) !== '.js' || file === 'info.js') continue;
+		if (file.substr(-3) !== '.js' || file === 'info.js' || file === 'github.js') continue;
```

Comment:
I think a chat plugin should have a way to specify that it isn't enabled and continue loading.